### PR TITLE
User link

### PR DESCRIPTION
**Pull request by tonsojun**

### CODE REVIEW

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -16,9 +16,13 @@ def edit
   end
 
   def create
-    @meeting = Meeting.new(meeting_params)
-
+    #current_user devise assigned variable to store current user.
+    @meeting = current_user.meetings.new(meeting_params)
+p '*' * 120
```
Comment (geocodinglife) [inline app/controllers/meetings_controller.rb]: Please! remove this code.

File: app/views/partials/_main_nav_bar.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="/">Navbar</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+
```
Comment (geocodinglife) [inline app/views/partials/_main_nav_bar.html.erb]: remove extra space

File: app/views/partials/_main_nav_bar.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <% else %>
+          <%= link_to "Sign in", new_user_session_path, class: "nav-link" %>
+          <%= link_to "Create Account", new_user_registration_path, class: "nav-link" %>
+          <% end %>
+          <% if user_signed_in? %>
+          <%= link_to "Create Appointment", meetings_path, class: "nav-link" %>
+            <% else %>
+          <%= link_to "Create Appointment", new_user_registration_path, class: "nav-link" %>
+          <% end %>
+
```
Comment (geocodinglife) [inline app/views/partials/_main_nav_bar.html.erb]: remove empty space

File: db/seeds.rb [hunk 4]
```diff
@@ -7,5 +7,31 @@
...
 
-Role.create([{name:"Admin", code:"admin"}, {name:"Default", code:"default"}, {name:"Physician", code:"physician"}])
+role = Role.create([{name:"Admin", code:"admin"}, {name:"Default", code:"default"}, {name:"Physician", code:"physician"}])
+User.create(email: "test@test.com", password: "123456", role_id: Role.find_by(code: "default"))
+physician = User.create(email: "test@test1.com", password: "123456", role_id: 3)
+physician2 = User.create(email: "test@test2.com", password: "123456", role_id: 3)
+physician3 = User.create(email: "test@test3.com", password: "123456", role_id: 3)
+(1..10).each do |number|
+  Room.create!(number: number)
+#puts number
```
Comment (geocodinglife) [inline db/seeds.rb]: Please remove comment.

**Review by geocodinglife — CHANGES_REQUESTED** [review]
please check the changes requested.

File: app/controllers/meetings_controller.rb (hunk 1 above)
Comment (tonsojun) [reply app/controllers/meetings_controller.rb]: Done.

File: app/views/partials/_main_nav_bar.html.erb (hunk 2 above)
Comment (tonsojun) [reply app/views/partials/_main_nav_bar.html.erb]: done

File: app/views/partials/_main_nav_bar.html.erb (hunk 3 above)
Comment (tonsojun) [reply app/views/partials/_main_nav_bar.html.erb]: Done.

**Pull request merged**